server/daemon/src/main.rs

```diff
@@ -1044,10 +1044,24 @@ async fn kanidm_main(
 
             let healthcheck_url = match &sopt.check_origin {
                 true => format!("{}/status", config.origin),
-                false => format!("https://{}/status", config.address),
+                false => {
+                    if config.address.starts_with("[::]") {
+                        format!(
+                            "https://{}/status",
+                            config.address.replace("[::]", "localhost")
```

Comment:
Why not just make it `[::1]`? Why replace the `[::1]` to localhost? Seems like we should only need to worry about the `[::]`

Reply:
Testing it now again locally, it works fine either way so I... must have been doing something weird the other day... going back to OP on #3002 to find out what's going on there.

Reply:
The only reason to fallback to localhost is to allow swapping over to 127.0.0.1